demo/laneLoader.js

```diff
@@ -528,21 +548,78 @@ function updateSegments(laneSegments, clonedBoxes, prevIsContains, point, index,
   return newIsContains
 }
 
+
+// Adds anomaly annotations
+function addAnnotations(laneGeometries) {
+  let point;
+
+  let aRoot = viewer.scene.annotations;
+  let aAnomalies = new Potree.Annotation({
+    title: "Lane Anomalies",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  aAnomalies.visible = false;
+  aRoot.add(aAnomalies);
+  let aLeft = new Potree.Annotation({
+    title: "Left",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  let aRight = new Potree.Annotation({
+    title: "Right",
+    position: laneGeometries.rightAnomalies[0]
+
+  });
+  aAnomalies.add(aLeft);
+  aAnomalies.add(aRight);
+
+  // Add anomalies layer
+  for (let ii=0, len=laneGeometries.leftAnomalies.length; ii<len; ii++) {
+    point = laneGeometries.leftAnomalies[ii];
+    let aAnomaly = new Potree.Annotation({
```

Comment:
`const`, rather than `let`, where appropriate. Then I don't have to look to see if this is some state that is changed somewhere.

[Please make this change everywhere.]
[I'm agnostic about const vs. let for things like vectors or dictionaries where the name itself isn't assigned, but the value is changed, via push or other methods.]